File: decode/decode.cpp
#include <iostream>
#include <string>
#include <stdio.h>

using namespace std;

int main(){
        // put your video name here
        string videoname="samplevideo.mp4";
        string cmd = "/opt/decode.sh -i "+videoname+" -o ./outputfile1";
        system(cmd.c_str());
        return 0;
}
 

File: decode/decode.py
#!/usr/bin/env python3

'''
Simple example to demonstrate HW accelerated decoding using python on BrainyPi.

Note: Do not use this code in production.
'''

import subprocess

# Put input video name here
videoname = "samplevideo.mp4"

a= subprocess.call(['/opt/decode.sh','-i',videoname,'-o','./outputfile1']) 
